xml/common_intro_feedback_i.xml

```diff
@@ -10,69 +10,43 @@
  <title>Feedback</title>
 
  <info>
-      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
-        <dm:maintainer>tbazant@suse.com</dm:maintainer>
-        <dm:status>editing</dm:status>
-        <dm:deadline/>
-        <dm:priority/>
-        <dm:translation>yes</dm:translation>
-        <dm:languages/>
-        <dm:release>SES 6</dm:release>
-      </dm:docmanager>
-    </info>
-    <para>
+   <dm:maintainer>tbazant@suse.com</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>yes</dm:translation>
+   <dm:languages/>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+
+ <para>
   Several feedback channels are available:
  </para>
 
  <variablelist>
-  <varlistentry os="osuse">
-   <term>Bugs and Enhancement Requests</term>
-   <listitem>
-    <para>
-     To report bugs for a product component, or to submit enhancement requests,
-     please use <link xlink:href="https://bugzilla.novell.com/"/>. For
-     documentation bugs, submit a bug report for the component
-     <guimenu>Documentation</guimenu> of the respective product.
-    </para>
-    <para>
-     If you are new to Bugzilla, you might find the following articles helpful:
-    </para>
-    <itemizedlist mark="bullet" spacing="normal">
-     <listitem>
-      <para>
-       <link xlink:href="http://en.opensuse.org/openSUSE:Submitting_bug_reports"/>
-      </para>
-     </listitem>
-     <listitem>
-      <para>
-       <link xlink:href="http://en.opensuse.org/openSUSE:Bug_reporting_FAQ"/>
-      </para>
-     </listitem>
-    </itemizedlist>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;sled;slepos">
+  <varlistentry>
    <term>Bugs and Enhancement Requests</term>
    <listitem>
     <para>
      For services and support options available for your product, refer to
      <link xlink:href="http://www.suse.com/support/"/>.
     </para>
     <para>
-     To report bugs for a product component, log in to the &ncc; from
-     <link xlink:href="http://www.suse.com/support/"/> and select <menuchoice>
-     <guimenu>My Support</guimenu> <guimenu>Service Request</guimenu>
-     </menuchoice>.
+     To open a service request, you need a subscription at SUSE Customer
+     Center. Go to <link xlink:href="https://scc.suse.com/support/requests"/>,
+     log in, and click Create New.
```

Comment:
Shouldn't there be some kind of markup or at least quotation marks around "Create New"?